Inventory/Controllers/APIController.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using InventoryLogic.Facade;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace InventoryAPI.Controllers
+{
+    [Route("api/v{v:apiVersion}/[controller]")]
+    [ApiController]
+    [ApiVersion("0.1")]
+    public class APIController<Type> : ControllerBase
```

Comment:
het moet een abstracte klasse zijn die geïmplementeerd moet worden lijkt mij.

Reply:
de default abstracte implementaties met een IFacade interface is wel mooi 👍

Reply:
Klopt, ik voeg abstract toe. En thanks!